mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_Mobile.java

```diff
@@ -522,7 +572,7 @@ public void setCurrentPane(final Pane pane) {
     }
 
     public void switchToPane(final Pane pane) {
-        if (mDesktopMode) return;
+        if ((mDesktopMode) || (mDexRunning && mDexDesktopMode)) return;
```

Comment:
The same condition repeats in different parts of code.
Shouldn't it be separate function?

Reply:
I added the Dex code to where it was already being determined for the original Desktop, if I put it to a separate function should I pull the entire if statement?